### PR TITLE
Added tooltip for signing the proposal

### PR DESCRIPTION
**Pull request by amichard**
#397 #398 #399

Added tooltips for the disabled Sign 1/2 and Sign 2/2 to give hints to the user on why it's disabled. Also, changed the breadcrumbs colouring.

Changelog:
- Sign 1/2 and Sign 2/2 should now be visible to Credit Trader roles
- Credit Traders will see a tooltip when hovering over the Sign 1/2 and Sign 2/2 buttons
- Reversed the colouring for the breadcrumbs
- Changed Recommended/Not Recommended to Reviewed

### CODE REVIEW

File: frontend/src/constants/routes.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const NOTIFICATIONS = '/notifications';
 export const SETTINGS = '/settings';
 
 // API Routes
-export const BASE_URL = `${window.location.protocol}//${window.location.host}/api`;
+export const BASE_URL = `${window.location.protocol}//${window.location.host}/api-business`;
```
Comment (plasticviking) [inline frontend/src/constants/routes.js]: Accidental inclusion?

Comment (amichard) [reply frontend/src/constants/routes.js]: Oops. Yes. Good catch!

**Comment by amichard** [comment]
[Breadcrumbs | Change Recommended breadcrumb to Reviewed |1|](https://trello.com/c/xCbhUjJU/659-breadcrumbs-change-recommended-breadcrumb-to-reviewed-1)

**Comment by amichard** [comment]
[Credit Transfer | Create Information text boxes for non-clickable Sign 1/2 and Sign 2/2 buttons |2|](https://trello.com/c/wvxQiY7p/680-credit-transfer-create-information-text-boxes-for-non-clickable-sign-1-2-and-sign-2-2-buttons-2)

**Comment by amichard** [comment]
[Breadcrumbs | Reverse breadcrumb color scheme |2|](https://trello.com/c/njOKLpae/681-breadcrumbs-reverse-breadcrumb-color-scheme-2)

**Pull request merged**